ros_ws/src/teleoperation/include/joystick_map.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <sensor_msgs/Joy.h>
+
+class JoystickMap
+{
+    protected:
+    JoystickMap()
+    {
+    }
+
+    public:
+    virtual ~JoystickMap()
+    {
+    }
+
+    /**
+     * @brief Get value of the steering axis. -1 is maximal left, 1 is maximal right
+     *
+     * @return float
+     */
+    virtual float getSteeringAxis(const sensor_msgs::Joy::ConstPtr& joystick) = 0;
```

Comment:
The method name and summary don't match. Does this return the steering axis or its value? Does axis refer to the car or the joystick? It should probably be named `getSteering`.